tekton/pipeline_run_test.go

```diff
@@ -262,4 +269,58 @@ var _ = Describe("PipelineRun", func() {
 			})
 		})
 	})
+
+	Context("When calling getPipelineRef", func() {
+		It("should return a PipelineRef without resolver if the releaseStrategy contains no bundle", func() {
+			releaseStrategy := &v1alpha1.ReleaseStrategy{
+				Spec: v1alpha1.ReleaseStrategySpec{
+					Pipeline:              "release-pipeline",
+					Bundle:                "",
```

Comment:
`Bundle` is optional, I think omitting the field is the same as putting the empty string (and is closer to what a user would actually do so we are testing like a user)

Reply:
Thanks for highlighting, @johnbieren  Fixed.